bugfix

naumvd95 opened the pull request:
remove unnecessary checkers
codestyle-fixes

naumvd95 committed:
bugfix

remove unnecessary checkers
codestyle-fixes

rkhozinov commented on servers/partner-ci/builders/upgrade.test.sh:
```diff
@@ -26,10 +26,10 @@ fi
-  version=$(grep "PLUGIN_VERSION" < build.plugin_version | cut -d= -f2 )
-  export PLUGIN_VERSION=${version:?}
-fi
+#if [ "${SNAPSHOTS_ID}" != "released" ]; then
```
remove that commented part

rkhozinov commented on servers/partner-ci/builders/upgrade.test.sh:
```diff
@@ -44,11 +44,10 @@ export VENV_PATH="${HOME}/${FUEL_RELEASE:?}-venv"
    -k                                 \
    -K                                 \
-   - w "{$WORKSPACE}"                 \
+   -w "$WORKSPACE"                   \
```
remove a space after `"$WORKSPACE"`

rkhozinov commented on servers/partner-ci/utils/utils.yaml:
```diff
@@ -148,7 +148,7 @@
     scm:
       - git:
-          url: https://github.com/rkhozinov/jenkins-jobs.git
+          url: https://github.com/rkhozinov/jenkins-jobs
```
separate this change from previous changes.
they don’t related, so make different prs

naumvd95 committed:
remove unnesecary redefining vars

restructure inject steps

rkhozinov committed:
code-style fixes

rkhozinov committed:
code-style fixes

utility-jobs: changes associated with this jobs will be replaced into
another PR

naumvd95 committed:
merge

rkhozinov committed:
Merge pull request #104 from rkhozinov/merge-jobs

merging all jobs to sync new changes (see change-description in each commit)
